content/knowledge_base/glossary.md

```diff
@@ -0,0 +1,141 @@
+# Glossary
+
+## General Blockchain Concepts
+
+### Consensus
+
+TODO: add
+
+### Blockchain
+
+TODO: add
+
+### Gossip Algorithms
+
+TODO: add
+
+### Liveness Threshold
+
+The number of malicious participants that can be tolerated before the consensus protocol stops functioning.
+
+### Nakamoto Type of Consensus
+
+TODO: introduce (for BTC)
+A consensus algorithm that provides a probabilistic safety guarantee; its decisions may revet with some probability ε.
+A permissionless protocol allowing each node to join or leave the system at any time.
+TODO: finish
+
+### Safety
+
+TODO: add
+
+## Avalanche Concepts
+
+### Ancestor Set
+
+TODO: add
+
+### Chit
+
+A transaction receives a chit (boolean value) when the majority (quorum) of queried nodes respond with an approval for that transaction.
+
+### Confidence
+
+The sum of transaction/vertex's decendants cihts and its own chit.
+The chit value is a result of a one-time query for its associated transaction and becomes immutable afterwards.
+Because chit values can be 0 or 1 - c ∈ {0,1} confidence values are monotonic.
+
+### Consecutive Success Number
+
+The number of times a transaction or its decendant received an approval from majority (quorum) of queried nodes.
+TODO: add more thorough description
+
+### Progeny
+
+TODO: add
+
+### Slush
+
+TODO: add
+
+## Other
+
+### Graph
+
+TODO: introduce the idea in one sentence without technicals
+
+A graph is pair of sets G = (V, E).
+V is a set of elements called vertices (singular: vertex) and E is a set of paired vertices and its elements are called edges.
+An edge {x, y} contains vertices x and y and they are its endpoints.
+A vertex may belong to no edge and in that case it is not joined to any other vertex.
+The set of vertices in a graph is a discrete value, meaning the set of edges is also a discrete value.
+The number of vertices - |V| is the "order" of the graph and the number of edges - |E| is the "size" of the graph.
+
+### Directed Graph
+
+A directed graph comprises of set of vertices connected by "directed" edges - also called arcs.
```

Comment:
```suggestion
A directed graph comprises of set of vertices connected by "directed" edges — also called arcs.
```

Tip: The em-dash can be written easily by typing `ALT+0151` :p Very useful for large documents like this hehe.

Reply:
Corrected https://github.com/optakt/optakt.github.io/pull/13/commits/48a7f92c1e451ec49d7b7f2cb79fa55a85a83a48#diff-55a59bbede67c988e24e88ffb8a7b56d20454b9ab95a3fd8567e790585338dfbR120
I didn't know that, will be useful to me, Thanks!